finality-aleph/src/environment.rs

```diff
@@ -131,18 +131,18 @@ impl<B: Block, H: Hash> UnitStore<B, H> {
 pub(crate) struct Environment<B: Block, H: Hash, C, BE, SC> {
     client: Arc<C>,
     select_chain: SC,
-    tx_consensus: Option<mpsc::UnboundedSender<NotificationIn<H>>>,
-    rx_consensus: Option<mpsc::UnboundedReceiver<NotificationOut<H>>>,
+    tx_consensus: mpsc::UnboundedSender<NotificationIn<H>>,
+    rx_consensus: mpsc::UnboundedReceiver<NotificationOut<H>>,
     tx_network: mpsc::UnboundedSender<NetworkCommand<B, H>>,
     rx_network: mpsc::UnboundedReceiver<NetworkEvent<B, H>>,
-    rx_order: Option<tokio::sync::mpsc::UnboundedReceiver<Vec<H>>>,
```

Comment:
There is no better moment to initialize fields than in the constructor

Reply:
Agreed, this was an artifact from times we thought environment will be a shared struct.